plugins/by-name/aerial/default.nix

```diff
@@ -0,0 +1,647 @@
+{ lib, ... }:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+  inherit (lib) types;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "aerial";
+  originalName = "aerial.nvim";
+  package = "aerial-nvim";
+
+  maintainers = [ lib.maintainers.GaetanLepage ];
+
+  settingsOptions =
+    let
+      size = with types; either ints.unsigned (numbers.between 0.0 1.0);
+      listOfSize = with types; either size (listOf size);
```

Comment:
Is this always supposed to be a 2-elem list? Should we be using our custom `listOfLen` type?

Reply:
Also, this has no raw-lua support